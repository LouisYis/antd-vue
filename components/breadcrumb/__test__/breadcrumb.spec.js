import Vue from 'vue'
import BreadCrumb from '../index'

const Constructor = Vue.extend(BreadCrumb)
const Instance = new Constructor({
  propsData: {
    routes: ['home', 'hello'],
    separator: '>'
  }
}).$mount()

describe('BreadCrumb component', () => {
  test('component should match snapshot', () => {
    expect(Instance.$el).toMatchSnapshot()
  })
})
