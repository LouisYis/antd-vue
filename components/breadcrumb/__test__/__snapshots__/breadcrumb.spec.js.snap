// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BreadCrumb component component should match snapshot 1`] = `
<div
  class="vk-breadcrumb"
>
  <span
    class="vk-breadcrumb-item"
  >
    <span
      class="vk-breadcrumb-item-name"
    >
      home
    </span>
    <span
      class="vk-breadcrumb-separator"
    >
      >
    </span>
  </span>
  <span
    class="vk-breadcrumb-item"
  >
    <span
      class="vk-breadcrumb-item-name"
    >
      hello
    </span>
    <span
      class="vk-breadcrumb-separator"
    >
      >
    </span>
  </span>
</div>
`;
